clientv3/balancer.go

```diff
@@ -205,6 +205,14 @@ func (b *simpleBalancer) Close() error {
 	return nil
 }
 
+func getScheme(ep string) string {
```

Comment:
why not change `parseEndpoints` to return the scheme string instead of a bool instead of having a new function?